lib/IDE/Refactoring.cpp

```diff
@@ -4073,21 +4024,22 @@ FuncDecl *isOperator(const BinaryExpr *BE) {
 ///   completion(nil, MyError.Bad) // Result = [MyError.Bad], IsError = true
 /// }
 class HandlerResult {
```

Comment:
Maybe we should just store the ArgList with the range used instead? This is fine as is for now though I think. It'd just be nice to avoid creating the vectors where we don't really need them.

Reply:
Hmm, IMO the vector isn't too bad as the number of elements should be small in most cases, plus it's nice being able to expose an `ArrayRef`, and it would also give us the flexibility to support non-contiguous argument ranges if we needed to, what do you think?

Reply:
Yeah, definitely wouldn't worry about it for now even if we do end up changing it. This is the most 1:1 change 👍